lib/cli.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-const fs = require('fs');
+const TreeSync = require('tree-sync');
 const copyDereferenceSync = require('copy-dereference').sync;
```

Comment:
Can you swap the remaining usage of `copy-dereference` to use `tree-sync` (I believe it is in `build` command below)? Then we can remove it as a dependency...